asf_heat_pump_suitability/config/README.md

```diff
@@ -16,6 +16,7 @@
 | `["data_source"]["EW_census_land_area"]`                       | `source_data/2021_vMar2021_census_landareaKM_EW.csv`                                           | [ONS](https://geoportal.statistics.gov.uk/datasets/a488cb8fc9a74accb63cb52961e456ef/about)                                                          | March 2021  | This csv file contains the Standard Area Measurements (SAM) for the 2021 Statistical Areas in England and Wales as at Census Day (21 March 2021).                                                                                                                                                                                          |
 | `["data_source"]["UK_spa_offgasgrid"]`                         | `source_data/2024_vMar2024_SPA_offgaspostcode_UK.xlsx`                                         | [SPA](https://www.xoserve.com/help-centre/supply-points-metering/supply-point-administration-spa/)                                                  | March 2024  | The Off Gas Postcode Register is a list of postcodes where Xoserve holds no record of an active gas connection by either large or small gas transporters.                                                                                                                                                                                  |
 | `["data_source"]["E_historicengland_listed_buildings"]`        | `source_data/Jun2024_vJul2024_HistoricEngland_listedbuilding_E.gpkg`                           | [Historic England](https://opendata-historicengland.hub.arcgis.com/datasets/historicengland::national-heritage-list-for-england-nhle/about?layer=3) | June 2024   | Listed buildings in England are structures protected for their historical, architectural, or archaeological significance, as recorded in the National Heritage List for England (NHLE). This dataset, using the British National Grid (EPSG:27700) spatial reference, includes the geographic locations and boundaries of these buildings. |
+| `["data_source"]["E_historic_england_conservation_areas"]`     | `source_data/Aug2024_historic_england_conservation_areas_E.geojson`                            | [Historic England](https://www.planning.data.gov.uk/dataset/conservation-area)                                                                      | June 2024   | Conservation areas in England (places of special architectural and historic interest).                                                                                                                                                                                                                                                     |
```

Comment:
Here we just have to use the geojson because I could only find this file format (or CSV or JSON) for [the latest version of the cons area data](https://www.planning.data.gov.uk/dataset/conservation-area) that I could find. I couldnt find a .gpkg or .shp file for this version.

They do have geopackage and shapefile versions on the [Historic England website](https://opendata-historicengland.hub.arcgis.com/datasets/historicengland::conservation-areas/explore?location=52.734863%2C-2.491828%2C6.62), but this seems to be a slightly earlier version of the data (July 2024 instead of August 2024) and as I understand from the description, the gov data combines Historic England data with a couple of other sources. Ultimately I went with the gov data because I thought it had a few more cons areas. :)

Reply:
So the only reason is the file type availability, nothing to do with CRS in this case.

Reply:
Ok great to know, thanks for that! Just good to check I haven't misunderstood something and happy with your reasoning for going with the geojson.